### ISSUE TITLE
Should focus first focusable child

### ISSUE THREAD

**Issue by robdodson** [Polymer/core-overlay#63]
When a core-overlay is opened, focus should be moved to the first focusable child within the overlay. This is to prevent screen readers from interacting with background content.

ex: http://accessibility.oit.ncsu.edu/training/aria/modal-window/

**Comment by robdodson** [comment]
Even better explanation: https://www.marcozehe.de/2015/02/05/advanced-aria-tip-2-accessible-modal-dialogs/

**Comment by robdodson** [comment]
Derp, it helps to RTFM! It looks like this is covered by the `autofocus` attribute. I'll open a separate ticket for trapping tab.
